### PR TITLE
Added commands in exercises README.md to run ignored tests

### LINKED ISSUE #545: Ignored tests handling
Right now the README.md file in some exercises states that the only way to run ignored tests is to manually remove #[ignore] flag from test file. 

Perhaps README could also mention that you can run all ignored tests with `cargo test -- --ignored`
and any individual test with `cargo test test_name`?

Maybe also provide a [link](https://doc.rust-lang.org/book/second-edition/ch11-02-running-tests.html "Running tests in Rust") about testing along with help-page, modules and cargo links?

### PR DESCRIPTION
**Pull request by ZapAnton**
Implements suggestions from and closes #545.

### CODE REVIEW

File: config/exercise_readme.go.tmpl [hunk 1]
```diff
@@ -17,11 +17,28 @@ Execute the tests with:
 $ cargo test
 ```
 
-All but the first test have been ignored.  After you get the first test to
-pass, remove the ignore flag (`#[ignore]`) from the next test and get the tests
-to pass again.  The test file is located in the `tests` directory.   You can
-also remove the ignore flag from all the tests to get them to run all at once
-if you wish.
+All but the first test have been ignored. If you wish to run every possible test use:
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: Running `cargo test -- --ignored` does not always run every possible test; certain exercises have optional tests locked behind feature flags. Consider wording this as "If you wish to run all tests, use:".

File: config/exercise_readme.go.tmpl [hunk 2]
```diff
@@ -17,11 +17,28 @@ Execute the tests with:
...
+
+```bash
+$ cargo test -- --ignored
+```
+
+To run specific test, for example `some_test`, you can use:
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: You're missing an article. Consider wording this as "To run a specific test"

File: config/exercise_readme.go.tmpl [hunk 3]
```diff
@@ -17,11 +17,28 @@ Execute the tests with:
...
+
+```bash
+$ cargo test some_test
+```
+
+To find test names, view the tests source file wich is located in `tests` directory.
+Every function that contains `#[test]` flag above it's name is a separate test.
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: There should be no apostrophe in possessive "its".

File: config/exercise_readme.go.tmpl [hunk 4]
```diff
@@ -17,11 +17,28 @@ Execute the tests with:
...
+$ cargo test -- --ignored
+```
+
+To run specific test, for example `some_test`, you can use:
+
+```bash
+$ cargo test some_test
+```
+
+To find test names, view the tests source file wich is located in `tests` directory.
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: Missing an article. Consider wording as "located in the `tests` directory."

File: config/exercise_readme.go.tmpl [hunk 5]
```diff
@@ -17,11 +17,28 @@ Execute the tests with:
...
+Every function that contains `#[test]` flag above it's name is a separate test.
+If the specfic test is ignored use:
+
+```bash
+$ cargo test some_test -- --ignored
+```
+
+Alternatively you can edit the tests source file and remove the `#[ignore]` flag
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: This should remain the primary encouraged method of proceeding through the exercise. It's fine to list some alternatives, but iterative TDD is an Exercism standard.

Comment (coriolinus) [reply config/exercise_readme.go.tmpl]: Another possible method you might suggest might be to use a script:

```sh
for test in tests/*.rs; do
   sed -i -e '/#\[ignore\]/{
      s/#\[ignore\]\s*//
      /^\s*$/d
   }' $test
done
```

This addition is not required.

File: config/exercise_readme.go.tmpl [hunk 6]
```diff
@@ -35,6 +52,7 @@ If you want to know more about Exercism, take a look at the [contribution guide]
 [help-page]: http://exercism.io/languages/rust
 [modules]: https://doc.rust-lang.org/book/second-edition/ch07-00-modules.html
 [cargo]: https://doc.rust-lang.org/book/second-edition/ch14-00-more-about-cargo.html
+[rust-tests]: https://doc.rust-lang.org/book/second-edition/ch11-02-running-tests.html
```
Comment (coriolinus) [inline config/exercise_readme.go.tmpl]: Without including a markdown link like this `Documentation about [tests][rust-tests]` elsewhere in the readme, markdown processors will strip this link entirely. Please add such a markdown link somewhere.

**Comment by ZapAnton** [comment]
I think I managed to address every problem from your review. Could you give it another look?

**Comment by ZapAnton** [comment]
All right, I will then make one last commit to re-generate every README. 

You mentioned, that you would like to squash all commits after final iteration - should I do it on my side, or will you do it after merge?

**Comment by coriolinus** [comment]
Either way works for me. Thanks for regenerating!

**Comment by ZapAnton** [comment]
Didn't see any objections - perhaps this PR could be merged, @coriolinus?

**Pull request merged**